.github/workflows/build.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     strategy:
       matrix:
         vendor: [ msi ]
-        model: [ ms7d25_ddr4, ms7d25_ddr5, ms7e06_ddr4, ms7e06_ddr5 ]
```

Comment:
I know it is only for testing, but we need all platforms to be built.

Reply:
Got this accidentially into PR, was only to speed up tests.